### DISCUSSION TITLE
Why? Why make things harder?

### DISCUSSION THREAD

**Discussion by mursaleemomin** [npm/feedback#834] [Polls]
npm install gatsby-cli -g
npm WARN config global --global, --local are deprecated. Use --location=global instead.

Why change something that isn't bugging the community?
Why not focus on things that actually matter?
Why not learn from Yarn's principle - to make things easy for the community?
Why you do this?

E.g. It's like Vue & React:
Vue & Svelte communities - who want to be good successors to React, solving React's long-time issues intuitively like State Management & templating.
React community - rigid & wouldn't accept good solutions if they're coming from competing technologies. Instead of adopting & making things easy, introduces unintuitive solutions.

I see this everywhere in tech nowadays. It's not a competition. Adopt good solutions.

**Comment by ljharb** [comment]
It’s not. See the duplicate thread you filed here: https://github.com/npm/rfcs/discussions/663
